iron-form.html

```diff
@@ -56,6 +56,18 @@
       contentType: {
         type: String,
         value: "application/x-www-form-urlencoded"
+      },
+
+      /**
+       * By default, the form will display the browser's native validation
+       * UI (i.e. popup bubbles and invalid styles on invalid fields). You can
+       * manually disable this; however, if you do, note that you will have to
+       * manually style invalid *native* HTML fields yourself, as you are
+       * explicitly preventing the native form from doing so.
+       */
+      disableNativeValidationUi: {
```

Comment:
Do you suppose it's an anti-pattern to invert this, and force the user to _enable_ this behavior?